packages/arb-ts/integration_test/arb-bridge.test.ts

```diff
@@ -12,13 +15,55 @@ const preFundedWallet = new Wallet(preFundedSignerPK, ethProvider)
 const l1TestWallet = Wallet.createRandom(ethProvider)
 const l2TestWallet = Wallet.createRandom(arbProvider)
 
-const bridge = new Bridge(
-  erc20BridgeAddress,
-  'arb20BridgeAddress',
-  l1TestWallet,
-  l2TestWallet
-)
+let arb20BridgeAddress: string
+
+let bridge: Bridge | undefined
+
+describe('setup bridge', () => {
+  it('instantiate bridge ', async () => {
+    const ethBridge = EthERC20BridgeFactory.connect(
+      erc20BridgeAddress,
```

Comment:
I think this should be the inbox address, but you removed it from the config?

Reply:
you're right